static/js/lib/site_content.test.ts

```diff
@@ -263,6 +265,43 @@ describe("site_content", () => {
     })
   })
 
+  describe("main content and column UI", () => {
+    it("isMainContentField should return true when appropriate", () => {
+      Object.values(WidgetVariant).forEach(widget => {
+        const configField = makeWebsiteConfigField({ widget })
+        if (widget === WidgetVariant.Markdown) {
+          configField.name = MAIN_PAGE_CONTENT_FIELD
+          expect(isMainContentField(configField)).toBeTruthy()
+        } else {
+          expect(isMainContentField(configField)).toBeFalsy()
+        }
+      })
+    })
+
+    it("splitFieldsIntoColumns should split the main content field out from others", () => {
+      const fields: ConfigField[] = [
+        makeWebsiteConfigField({ widget: WidgetVariant.Text }),
+        makeWebsiteConfigField({ widget: WidgetVariant.Select }),
+        {
+          label:  "Body",
+          name:   MAIN_PAGE_CONTENT_FIELD,
+          widget: WidgetVariant.Markdown
+        },
+        makeWebsiteConfigField({ widget: WidgetVariant.Boolean })
+      ]
+      expect(splitFieldsIntoColumns(fields)).toEqual([
+        [
+          {
+            label:  "Body",
+            name:   MAIN_PAGE_CONTENT_FIELD,
+            widget: WidgetVariant.Markdown
+          }
```

Comment:
Nit: could you just use `fields[1]` here?

Reply:
yes! haha